src/Console/Marketplace/DownloadCommand.php

```diff
@@ -69,7 +69,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
             // If the plugin is already downloaded, refuse to download it again
             if (!$input->getOption('force') && is_dir(GLPI_MARKETPLACE_DIR . '/' . $plugin)) {
-                $error_msg = sprintf(__('Plugin "%s" is already downloaded. Use --force to force it to re-download.'), $plugin);
+                if (Controller::hasVcsDirectory($plugin)) {
+                    $error_msg = sprintf(__('Plugin "%s" as a local source versioning directory.'), $plugin);
+                    $error_msg .=  "\n" . __s('To avoid overwriting a potential branch under development, downloading is disabled.');
```

Comment:
```suggestion
                    $error_msg .=  "\n" . __('To avoid overwriting a potential branch under development, downloading is disabled.');
```